ros/resources/hosts.py

```diff
@@ -14,18 +14,18 @@ def get(self):
                     'id': '12345-57575757',
                     'account': '12345',
                     'vm_uuid': '12345a1',
-                    'state': 'Oversized',
-                    'recommendation_count': 4,
+                    'state': 'Crashloop',
+                    'recommendation_count': 5,
                     'organization_id': 1,
                     'performance_score': {
-                        'cpu_score': 70,
+                        'cpu_score': 20,
                         'memory_score': 20,
-                        'io_score': 80
+                        'io_score': 20
                     },
                     'facts': {
                         'provider': 'AWS',
```

Comment:
I think `cloud_provider` will be more appropriate key here. Also, on host inventory side key name is `cloud_provider`